web/src/games/checkers/game.ts

```diff
@@ -127,8 +127,8 @@ export function getValidMoves(G: IG, playerID: string, jumping?: ICheckerPieceWi
 }
 
 export function move(G: IG, ctx: Ctx, from: ICoord, to: ICoord): IG | string {
-  const fromCoord = coord(from.x, from.y);
-  const toCoord = coord(to.x, to.y);
+  const fromCoord = createCoord(from.x, from.y);
```

Comment:
Sorry, what I meat here is that you don't need the "fromCoord"and "toCoord" variables at all. You can just pass "from" and "to" directly to "toIndex" below.

Reply:
Done